e2e/test/iothub/TokenCredentialAuthenticationTests.cs

```diff
@@ -103,9 +103,15 @@ public async Task DigitalTwinClient_Http_TokenCredentialAuth_Success()
             // Call openAsync() to open the device's connection, so that the ModelId is sent over Mqtt CONNECT packet.
             await deviceClient.OpenAsync().ConfigureAwait(false);
 
+            var dtClientOptions = new DigitalTwinClientOptions
+            {
+                TokenCredentialAuthenticationScopes = IotHubAuthenticationScopes.DefaultAuthenticationScopes
```

Comment:
Is this necessary? I'd expect that the ```DigitalTwinClient.Create()``` call with no options would default to this value anyways

Reply:
Not necessary, probably belongs more in the sample. I was thinking in terms of how to demonstrate usage when I added this.

Reply:
I had a better idea to test the new overload. I am calling the new overload from the old one so even without this, it should be a good test now.